<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RED</class>
 <widget class="QWidget" name="RED">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RED</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_discovering" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_discovering">
        <property name="text">
         <string>Discovering Image Version...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="REDTabOverview" name="tab_overview">
      <attribute name="title">
       <string>Overview</string>
      </attribute>
     </widget>
     <widget class="REDTabSettings" name="tab_settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
     </widget>
     <widget class="REDTabProgram" name="tab_program">
      <attribute name="title">
       <string>Program</string>
      </attribute>
     </widget>
     <widget class="REDTabConsole" name="tab_console">
      <attribute name="title">
       <string>Console</string>
      </attribute>
     </widget>
     <widget class="REDTabVersions" name="tab_versions">
      <attribute name="title">
       <string>Versions</string>
      </attribute>
     </widget>
     <widget class="REDTabExtension" name="tab_extension">
      <attribute name="title">
       <string>Extension</string>
      </attribute>
     </widget>
     <widget class="REDTabImportExport" name="tab_importexport">
      <attribute name="title">
       <string>Import/Export</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>REDTabOverview</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_overview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabSettings</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_settings</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabProgram</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_program</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabConsole</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_console</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabVersions</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_versions</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabExtension</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_extension</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>REDTabImportExport</class>
   <extends>QWidget</extends>
   <header location="global">brickv.plugin_system.plugins.red.red_tab_importexport</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
